trunk/src/graphics/paraview/geom2vtk.F90

```diff
@@ -169,25 +163,28 @@ subroutine geom2vtk(Sname,Sfile, IcE,IcN,IcP)
       j = nobj_conf(ntype)
```

Comment:
this could use the array access from `NVERT` to be accelerated, do we actually need `nobj_conf` as its own function?

Reply:
The number of visualization vertices is not the same as the number of element vertices for second order elements (see `nobj_conf`)

Reply:
my bad, I missed seeing the change to `nobj_conf`